client/packages/system/src/Item/Components/StockItemSearchInput/hooks.ts

```diff
@@ -0,0 +1,33 @@
+import { useState, useCallback } from 'react';
+
+export const useOrganisationFilter = (searchText = '') => {
+  const filterBy = useCallback(
+    (value: string) => ({ name: { like: value } }),
+    [searchText]
+  );
+  const [filter, setFilter] = useState(filterBy(searchText));
+  return {
+    filter,
+    onFilter: (searchText: string) => setFilter(filterBy(searchText)),
+  };
+};
+
+export const usePagination = () => {
+  const [pagination, setPagination] = useState({
+    page: 0,
+    first: 1000,
```

Comment:
I thought that 1000 seemed pretty high, given that pagination is now in place. have added this as a prop and defaulted to 500 🤷

Reply:
Fair enough. 8e160ebd2